packages/cli/commands/import/credentials.ts

```diff
@@ -37,17 +44,20 @@ export class ImportCredentialsCommand extends Command {
 	};
 
 	async run() {
+		const logger = getInstance();
+		LoggerProxy.init(logger);
+
 		const { flags } = this.parse(ImportCredentialsCommand);
 
 		if (!flags.input) {
-			GenericHelpers.logOutput(`An input file or directory with --input must be provided`);
+			console.info(`An input file or directory with --input must be provided`);
 			return;
 		}
 
 		if (flags.separate) {
 			if (fs.existsSync(flags.input)) {
 				if (!fs.lstatSync(flags.input).isDirectory()) {
-					GenericHelpers.logOutput(`The paramenter --input must be a directory`);
+					console.info(`The paramenter --input must be a directory`);
```

Comment:
instead of using `console.log`, why don't we use Logger for all logs, and setup transports according to log level? Say info and lower (more important) go to file and standard output, and higher than info (debug etc) go to only file?

So that we have a consistent implementation?

Reply:
Hey Ahsan.

This was a dillema during development. In some cases, I only want messages to be output to terminal only (they don't really make sens to be added to a log file as this is an import credentials command).

So I was using both the console.* messages with LoggerProxy.* messages according to the type of message.

For a log file, my expectations as a developer is to always have all errors and even better, have debug messages on log files to help me solve issues. So actually log files should contain all relevant logs and UI prompts (even if the interface is the terminal) should not be there.

Does this make sense? Let me know your thoughts!